### ISSUE TITLE
Unsupported layer 2 items

### ISSUE THREAD

**Issue by donk7413** [kwsch/NHSE#502]
I don't know where is it and how to remove . I place an bridge kit from acnhspawner without take it from Tom. So now tom doesn't want i build anything and I have an unfinished bridge with cost of 0 so it's unbuildable. I try to remove it with the editor but I have this message : unsupported layer 2 items

**Comment by kwsch** [comment]
There are two layers in the field item editor. Delete the layer 2 items that are stacked on top of nothing.

The program is telling you that you haven't finished deleting stuff.

**Comment by donk7413** [comment]
I replace value of the bridge by 0, so it make it none. But when I import my save to the game, it say it's corrupted

**Comment by donk7413** [comment]
And I still have unsupported layer 2 item message

**Comment by kwsch** [comment]
Restoring a save file correctly is not related to this program, and I already told you how to remove unsupported layer 2 items.

**Comment by donk7413** [comment]
Its not about an item. Its about an impossible to finish bridge placed on map. I don't understand your explanation. " Delete the layer 2 items that are stacked on top of nothing". In map editor menu ? Which tab ?

**Comment by donk7413** [comment]
@berichan
I don't think i spamming anything. I have an issue, by using acnh spawner app. I try to fix it by using nhse editor. So i ask to his GitHub if I do correctly.
I also ask here if someone behave an fix for my issue, because it's concerning the acnh spawner app. 

Its rude to say I'm an spammer when I put my issue inn2 GitHub because I use theses two soft. 

I just ask the help of community, not about spamming like an kid. Because at the end, GitHub is for community project right ? And if I ask here, it's because I have try 4 or 5 way to remove the bridge without any success and it block my progression.

**Comment by kwsch** [comment]
GitHub is for reporting issues with a program, not for getting help using other tools like save restoring correctly.

The layer toggle is visible in the same editor window that alerts you that you have unsupported layer 2 items.
![image](https://user-images.githubusercontent.com/6393368/130329495-1acb0896-93e3-4aab-837f-3da5fdb9f692.png)

Clear the bridge data, delete items on layer 2 that don't have anything under them, and restore your save file correctly.

All these features work in NHSE. This is not an issue with the program.
